src/domain/useCase/deleteUnixFiles.go

```diff
@@ -12,31 +12,40 @@ import (
 const TrashDirPath string = "/app/.trash"
 
 type DeleteUnixFiles struct {
-	filesQueryRepo repository.FilesQueryRepo
-	filesCmdRepo   repository.FilesCmdRepo
+	filesQueryRepo        repository.FilesQueryRepo
+	filesCmdRepo          repository.FilesCmdRepo
+	activityRecordCmdRepo repository.ActivityRecordCmdRepo
 }
 
 func NewDeleteUnixFiles(
 	filesQueryRepo repository.FilesQueryRepo,
 	filesCmdRepo repository.FilesCmdRepo,
+	activityRecordCmdRepo repository.ActivityRecordCmdRepo,
 ) DeleteUnixFiles {
 	return DeleteUnixFiles{
-		filesQueryRepo: filesQueryRepo,
-		filesCmdRepo:   filesCmdRepo,
+		filesQueryRepo:        filesQueryRepo,
+		filesCmdRepo:          filesCmdRepo,
+		activityRecordCmdRepo: activityRecordCmdRepo,
 	}
 }
 
-func (uc DeleteUnixFiles) emptyTrash() error {
+func (uc DeleteUnixFiles) emptyTrash(
+	operatorAccountId valueObject.AccountId,
+	operatorIpAddress valueObject.IpAddress,
+) error {
 	trashPath, _ := valueObject.NewUnixFilePath(TrashDirPath)
 	err := uc.filesCmdRepo.Delete(trashPath)
 	if err != nil {
 		return err
 	}
 
-	return uc.CreateTrash()
+	return uc.CreateTrash(operatorAccountId, operatorIpAddress)
```

Comment:
I believe it's unnecessary to create separate trash bins for each account, let alone a DTO, since everything is handled within the method itself. You only need to provide the operator infos.